harvester/records/record.py

```diff
@@ -616,3 +616,46 @@ def get_single_subfield(tag: DataField, subfield: str) -> SubField | None:
             message = f"Multiple subfields found in tag for subfield: {subfield}"
             raise ValueError(message)
         return None
+
+    def get_single_tag_subfield_value(self, tag_code: str, subfield_code: str) -> str:
+        """Return single subfield value from a single tag."""
+        tag = self.get_single_tag(tag_code)
+        if not tag:
+            message = f"Record does not have single instance of tag '{tag_code}'"
```

Comment:
This phrasing is ambiguous. If `get_single_tag` would raise an error for many instances and this is intended to raise an error for 0 instances, I think `Record has no instances of tag` would be clearer

Reply:
Fair point!  Updating.